projects/pgai/pyproject.toml

```diff
@@ -15,7 +15,7 @@ dependencies = [
     "langchain-openai>=0.1,<1.0",
     "langchain-text-splitters>=0.2,<1.0",
     "pydantic>=2.0,<3.0",
-    "ollama>=0.4.5,<0.5.0",
+    "ollama>=0.3.0,<0.5.0",
```

Comment:
is litellm requiring an older version of ollama?

Reply:
No, I think this was a result of bad merge conflict resolution. Will fix.